src/NHN.DtoContracts/NHN.DtoContracts/Ofr/Data/HealthRegister.cs

```diff
@@ -102,17 +102,17 @@ public class HealthRegister
         public IList<PhysicalAddress> PhysicalAddresses { get; set; }
 
         /// <summary>
-        /// Elektoniske adresser til eieren av registeret. I hovedsak er dette snakk om en Hjemmeside representert av kodeverdien E_URL, 
-        /// og en Adresselenke for innsynsrett representert av kodeverdien E_PTL 
-        /// Kodeverk: <see href="/CodeAdmin/EditCodesInGroup/type_adressekomponeneter">type_adressekomponeneter</see> (OID 9044).
+        /// Elektoniske adresser til eieren av registeret. I hovedsak er dette snakk om en Hjemmeside representert av kodeverdien E_URL,
```

Comment:
Rette en trykkfeil til når vi først er innom :) 
```suggestion
        /// Elektroniske adresser til eieren av registeret. I hovedsak er dette snakk om en Hjemmeside representert av kodeverdien E_URL,
```